package.json

```diff
@@ -34,7 +34,7 @@
     "@opendollar/sdk": "1.7.0-rc.5",
     "@opendollar/svg-generator": "1.0.4",
     "@react-spring/web": "^9.7.3",
-    "@sentry/cli": "^2.31.2",
+    "@sentry/cli": "^2.31.0",
```

Comment:
Why the downgrade?

Reply:
It comes from #436 branch, I think we can discuss it later